examples/http-client-header-parameter/http_client_header_parameter.md

```diff
@@ -1,6 +1,8 @@
 # HTTP client - Header parameter
 
-The http module provides support for sending outbound request headers as resource method arguments. The headers can be provided as `map<string|string[]>` to the resource method. 
+The `http:Client` supports sending outbound request headers along with the request payload.
+This headers can be passed as an argument in the client resource method call. The headers should be provided as a `map` where the keys represent the header names and the entries represent the header values. The header values can be `string` or `string[]`.
+Use these headers to provide information about the request context, so that the server can tailor the response.
```

Comment:
Use this when you want to send additional headers as part of the request.

Reply:
Fixed via [519200e](https://github.com/ballerina-platform/ballerina-distribution/pull/3763/commits/519200e50bec0a96466340292b820dc4e4810c48)